Sources/Apodini/Properties/Published.swift

```diff
@@ -1,40 +1,42 @@
 /// Property wrapper that can be used to annotate properties inside of `ObservableObject`s.
 /// The `ObservableObject` will notify its subscribers if a `Published` property changes.
 @propertyWrapper
-public struct Published<Element>: Property {
-    private var wrapper: Wrapper<Element>
-    private var wrappedValueDidChange: Wrapper<(() -> Void)?>
+public class Published<Element>: Property {
```

Comment:
Doesn't changing this property wrapper to a class violate the condition that all objects conforming to `Property` have to be a struct? However, I agree that it makes sense to declare it as a class.

Reply:
You're right, `Published` shouldn't even be a `Property` as it isn't handled using the functionality of `Traversable.swift`.

Reply:
fixed with 4ef4e85